Example/Podfile.lock

```diff
@@ -9,8 +9,8 @@ EXTERNAL SOURCES:
     :path: "../"
 
 SPEC CHECKSUMS:
-  lottie-ios: 55a2de40418f8de2b155efaba4d4d1a025a5c570
+  lottie-ios: 364ab2a5168e95ca4f0775b967fded860428ea04
 
 PODFILE CHECKSUM: d7d1d3df3df6c9862ab67dbbfb2f695895dc9d7b
 
-COCOAPODS: 1.6.1
+COCOAPODS: 1.5.3
```

Comment:
we should probably have a Gemfile in here to prevent these versions from changing so much

Reply:
It's really no big deal since its just the example project. @Coeur